### PR TITLE
Fix EZP-26798: Query parameters get lost when being redirected with m…

### PR DESCRIPTION
**Pull request by blankse**
…obile device detect filter

JIRA: https://jira.ez.no/browse/EZP-26798

### CODE REVIEW

File: kernel/private/classes/ezpmobiledeviceregexpfilter.php [hunk 1]
```diff
@@ -133,7 +133,7 @@ public function redirect()
                 }
```
Comment (yannickroger) [inline kernel/private/classes/ezpmobiledeviceregexpfilter.php]: Should this redirect be also updated?

Comment (blankse) [inline kernel/private/classes/ezpmobiledeviceregexpfilter.php]: Yes, you're right. I updated it.

**Comment by blankse** [comment]
@bdunogier @andrerom Can you review it?

**Comment by yannickroger** [comment]
Thanks @blankse

**Pull request merged**